mix.exs

```diff
@@ -12,7 +12,7 @@ defmodule Appsignal.Phoenix.MixProject do
         licenses: ["MIT"],
         links: %{"GitHub" => "https://github.com/appsignal/appsignal-elixir-phoenix"}
       },
-      elixir: "~> 1.9",
+      elixir: "~> 1.11",
```

Comment:
I don't think we want to bump our Elixir requirement. Our main version requires 1.9 still and also uses Logger.warning.
In our latest package we have a [`Appsignal.Utils.warning/1` helper function](https://github.com/appsignal/appsignal-elixir/blob/b1ffe29180080cc0bf871e69c632b6b780da8f60/lib/appsignal/utils.ex#L26C1-L30) that will handle the warning/warn function call for the different versions.